specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-05-01-preview/common/EntityTypes.json

```diff
@@ -1000,6 +1010,87 @@
           "readOnly": true,
           "type": "array",
           "x-ms-identifiers": []
+        },
+        "nicEntityIds": {
```

Comment:
Without context, I don't know what format these Ids have, are these Azure resource ids? If yes, how are you handling RBAC?